_content/setup-wallets.md

```diff
@@ -0,0 +1,79 @@
+
+If you want to send actual value in your app with an Interledger account, refer to the [Rust implementation of Interledger](http://interledger.rs), or the Java implementation of Interledger at [Hyperledger Quilt](https://www.hyperledger.org/projects/quilt). Because these efforts are open source, your contributions are welcome.
+
+### Log in to Xpring Testnet to create an Interledger account
+
+1. Open the Xpring Testnet login page: [https://wallet.ilpv4.dev/](https://wallet.ilpv4.dev/)<br>
+   Read the Terms of Use and the Privacy Policy, as linked on that page.
+2. If you agree with the specified terms and privacy policy, click **Sign in**.<br>
+   You are prompted to sign in with one of Facebook, Github, or Google credentials.
+3. For your login, select the option you prefer.<br>
+   You can create multiple accounts that you control by signing in separately with other options.
+4. You might be prompted to log in with your credentials, or if you are already logged in, you might be prompted for further authentication.<br>
+   When you are logged in, you see a page with your payment pointer, in the format `$money.ilpv4.dev/<your-account-name>`.
+   As indicated by the payment pointer, this wallet uses the Interledger Protocol, v4.
+   Because a new account does not have any test XRP value, you will want to add value to it.
+5. Click **Make it rain**.<br>
+  Your account now has 10 test XRP. To add more test XRP, continue to click the button until you have as many test XRP as you want.
+6. If you want someone to send you test XRP, provide them with your payment pointer.
+7. If you want to send test XRP to someone, request that they send you their payment pointer.<br>
+   You can readily set up multiple accounts to get different payment pointers, and therefore experiment with sending test XRP in different scenarios where you have ready access to the accounts on both sides.
+
+When you subsequently log in to Xpring Testnet with the same credentials, you will see the current state of your wallet with the current amount of test XRP.   
+
+#### Send money with your Xpring login wallet
+
+When you have test XRP in your account, you can practice sending test XRP to another account.
+
+1. Sign in to Xpring Testnet at [https://wallet.ilpv4.dev/](https://wallet.ilpv4.dev/). Look at the **Send** portion of the page on the right.
+2. Enter the number of test XRP, in integer form, that you want to send.
+3. Enter the payment pointer for the address to which you want to send test XRP.
+4. Click **Send**.
+
+The amount of XRP in the lower left adjusts to the new value. To add more value to your account, click **Make it rain**.
+
+### Create an account at rafiki.money
+
+The [rafiki.money](https://rafiki.money) site is an implementation of Rafiki, which is an Interledger connector. You can [read more](https://medium.com/interledger-blog/introducing-rafiki-e3de4710d3de) about why Rafiki was created.
```

Comment:
Yeah, we should just emphasize that https://rafiki.money is an Interledger-enabled wallet. The link should just be to rafiki.money -- there's not really anything more to read here.

@adrianhopebailie @matdehaast for their opinion, since they created rafiki.money